spec/models/group_assignment_spec.rb

```diff
@@ -44,6 +44,22 @@
     end
   end
 
+  describe 'exclusion of reserved words from title' do
+    let(:organization) { create(:organization) }
+
+    it 'ensures RecordInvalid error when GroupAssignment title is \'new\'' do
+      expect do
+        create(:group_assignment, organization: organization, title: 'new')
+      end.to raise_error(ActiveRecord::RecordInvalid, 'Validation failed: Your assignment title is reserved')
+    end
+
+    it 'ensures RecordInvalid error when GroupAssignment title is \'update\'' do
+      expect do
+        create(:group_assignment, organization: organization, title: 'update')
+      end.to raise_error(ActiveRecord::RecordInvalid, 'Validation failed: Your assignment title is reserved')
+    end
+  end
```

Comment:
I apologize but this is leftover from my other branch. Removing this in favor of id based routing.